airbyte-integrations/connectors/source-bing-ads/unit_tests/test_source.py

```diff
@@ -28,12 +28,6 @@ def logger_mock_fixture():
 @patch.object(source_bing_ads.source, "Client")
 def test_streams_config_based(mocked_client, config):
     streams = SourceBingAds().streams(config)
-    assert len(streams) == 15
-
-
-@patch.object(source_bing_ads.source, "Client")
-def test_streams_all(mocked_client):
```

Comment:
Why this part deleted?

Reply:
Two functions (test_streams_config_based  and test_streams_all) actually perform one task - to calculate the number of reports that will be executed as a result. Since the user will no longer be able to customize the choice of periodic reports, there will always be 25 of them. In fact, this is now the base implementation. Therefore, there is no need to do it 2 times.